packages/appium/test/fixtures/args-for-schema.js

```diff
@@ -185,9 +185,9 @@ export default [
     },
   ],
   [
-    ['--plugins'],
+    ['--use-plugins'],
```

Comment:
@boneskull is this file used anymore? i was surprised that whatever it was for didn't break since it missed the change to the new arg.

Reply:
Hmm.. yes it looks like it can be removed.